substrate/frame/merkle-mountain-range/src/lib.rs

```diff
@@ -90,11 +87,11 @@ mod tests;
 /// crate-local wrapper over [frame_system::Pallet]. Since the current block hash
 /// is not available (since the block is not finished yet),
 /// we use the `parent_hash` here along with parent block number.
-pub struct ParentNumberAndHash<T: frame_system::Config> {
-	_phantom: core::marker::PhantomData<T>,
+pub struct ParentNumberAndHash<T: Config> {
```

Comment:
Before it bounded only frame system config, this is a change.

Reply:
I'd assume it's unnecessary, please let me know if any side effects here.

Reply:
yes ok